src/useCases/listCourtCases.ts

```diff
@@ -15,6 +15,12 @@ const listCourtCases = async (connection: DataSource, queryParams: QueryParams):
     )
     .limit(queryParams.limit)
 
+  if (queryParams.defendantName) {
+    query.where("lower(courtCase.defendantName) like '%' || :name || '%'", {
```

Comment:
For case-insensitive search in Postgres, it seems we can use [ILIKE](https://www.postgresql.org/docs/9.0/functions-matching.html) keyword. So we don't need to use `lower()` function.

Reply:
Refactored in [b22d455](https://github.com/ministryofjustice/bichard7-next-ui/pull/24/commits/b22d455b17a38c3e89b99d3442fa71a1e35c2587)